### PR TITLE
Add kerning support

### PR DESCRIPTION
**Pull request by joewhitsitt**
Building upon #26 with @kartsims' requests, not trying to take credit away from @ilyabirman.

Would really love to see a Packagist release with this functionality!

### CODE REVIEW

File: src/EasySVG.php [hunk 1]
```diff
@@ -165,6 +176,15 @@ public function setFontSVG( $filepath ) {
...
                 }
+                if ($name == 'hkern') {
+                  $u1 = $this->_utf8ToUnicode($z->getAttribute('u1'));
+                  $u2 = $this->_utf8ToUnicode($z->getAttribute('u2'));
+                  if (isset($u1[0]) and isset ($u2[0])) {
+                    $k = $z->getAttribute('k');
+                    $this->font->hkern[$u1[0]][$u2[0]] = $k;
+                  }
+                }
+
```
Comment (kartsims) [inline src/EasySVG.php:187]: slight indentation issue here but I'll fix it

**Comment by kartsims** [comment]
Amazing job thanks for participating

I don't remember how to publish on Packagist but I'll give it a go :)

**Comment by kartsims** [comment]
OK I don't know how this works but I just had to click the public link on the package page! 

I couldn't associate this package to my packagist login. If you have any resource to point me to (so that I can manage versions betters, this should be a 1.1) I'd be happy to read them but at least the code is up-to-date now :)

**Comment by richardbporter** [comment]
> OK I don't know how this works but I just had to click the public link on the package page!
> 
> I couldn't associate this package to my packagist login. If you have any resource to point me to (so that I can manage versions betters, this should be a 1.1) I'd be happy to read them but at least the code is up-to-date now :)

Here are the docs on updating packages: https://packagist.org/about#how-to-update-packages

**Pull request merged**